### PR TITLE
Improve the management of Daikin Sensors and switch Mode

### PR DESCRIPTION
**Pull request by pipiche38**
Better management of Target Temperature and Humidity as well as managing orrectly the switch from one mode to an other one.

### CODE REVIEW

**Comment by pipiche38** [comment]
This PR addresses #2570 and #2565 as well as a closed #2562

File: hardware/Daikin.cpp [hunk 1]
```diff
@@ -125,7 +125,7 @@ bool CDaikin::StopHardware()
 
 void CDaikin::Do_Work()
 {
-	int sec_counter = Daikin_POLL_INTERVAL - 2;
+	sec_counter = Daikin_POLL_INTERVAL - 2; // Trigger immediatly (in 2s) a POLL after startup.
```
Comment (gizmocuz) [inline hardware/Daikin.cpp]: Could you change this variable so it starts with m_ as all class member variables should do?

File: hardware/Daikin.cpp [hunk 2]
```diff
@@ -260,7 +260,7 @@ void CDaikin::SetSetpoint(const int /*idx*/, const float temp)
 	szURL << "pow=" << m_pow;
 	szURL << "&mode=" << m_mode;
 
-	// cibre température
+	// cible température
 	char szTmp[100];
```
Comment (gizmocuz) [inline hardware/Daikin.cpp:264]: I think this is still French?

File: hardware/Daikin.h [hunk 3]
```diff
@@ -41,5 +41,9 @@ class CDaikin : public CDomoticzHardwareBase
 	std::string m_led;
 	volatile bool m_stoprequested;
 	std::shared_ptr<std::thread> m_thread;
+	int sec_counter;    // use in Do_Work() and will allow to trigger faster GetMeterDetails()        
+	std::string m_dt[8];     // Memorized Temp target for each mode. Retreived from Daikin device
```
Comment (gizmocuz) [inline hardware/Daikin.h]: This comment seems logical and miss it's purpose. Best to remove it

Comment (gizmocuz) [reply hardware/Daikin.h]: Retrieved from... Can be removed as this is a Daikin class

**Review by gizmocuz — COMMENTED** [review]
Thanks for the PR.
Some minor changes would be great.

**Comment by pipiche38** [comment]
Should be good now. Let see the integration check.

**Comment by gizmocuz** [comment]
Thank you !!

**Pull request merged**